seeq-link-sdk-debugging-agent/src/main/resources/data/configuration/link/MyCompany MyConnector.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Do we need this to be present in GIT?

It is not misleading for a connector developer to have it here?